greetservice-impl/bnd.bnd

```diff
@@ -0,0 +1 @@
+Import-Package: !java.*,*
```

Comment:
Is it checked that the manifest contains this package import and it works if it's explicitely excluded ?